sqlalchemy_redshift/dialect.py

```diff
@@ -102,7 +102,7 @@ def _get_relation_key(name, schema):
 def _get_schema_and_relation(key):
     if '.' not in key:
         return (None, key)
-    identifiers = SQL_IDENTIFIER_RE.findall(key)
+    identifiers = re.split("\.", key, 1)
```

Comment:
Sorry I missed this comment. Yeah, you're right. This does not handle delimited identifiers well. Let me have a look.